app/src/main/res/layout/fragment_add_login.xml

```diff
@@ -111,8 +112,8 @@
         android:colorControlActivated="?attr/textPrimary"
         android:textColor="?attr/textPrimary"
         android:contentDescription="@string/saved_login_username_description"
-        app:layout_constraintEnd_toEndOf="@id/usernameHeader"
-        app:layout_constraintStart_toStartOf="@id/usernameHeader"
+        app:layout_constraintEnd_toEndOf="parent"
```

Comment:
as we discussed 🙂
```
android:id="@+id/inputLayoutUsername"           
android:layout_width="0dp"
```

Reply:
Thanks for pointing this out, I fixed it